src/Microsoft.PowerShell.MarkdownRender/ListItemBlockRenderer.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using Markdig;
+using Markdig.Syntax;
+using Markdig.Renderers;
+
+namespace Microsoft.PowerShell.MarkdownRender
+{
+    /// <summary>
+    /// Renderer for adding VT100 escape sequences for items in a list block.
+    /// </summary>
+    internal class ListItemBlockRenderer : VT100ObjectRenderer<ListItemBlock>
+    {
+        protected override void Write(VT100Renderer renderer, ListItemBlock obj)
+        {
+            var parent = obj.Parent as ListBlock;
+
+            if (parent != null)
+            {
+                if (!parent.IsOrdered)
+                {
+                    foreach (var line in obj)
+                    {
+                        RenderWithIndent(renderer, line, parent.BulletType, 0);
+                    }
+                }
+            }
+        }
+
+        private void RenderWithIndent(VT100Renderer renderer, MarkdownObject block, char listBullet, int indentLevel)
+        {
+            // Indent left by 2 for each level on list.
+            string indent = "".PadLeft(indentLevel * 2);
```

Comment:
We should use a cache for the space string. 
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/utils/StringUtil.cs#L69

Reply:
fixed.